Xamarin.PropertyEditing.Windows/BrushEditorControl.cs

```diff
@@ -22,9 +22,12 @@ public override void OnApplyTemplate ()
 			this.brushTabs = this.brushBoxPopup?.Child?.GetDescendants<BrushTabbedEditorControl>().FirstOrDefault();
 
 			if (IsEnabled && this.brushBoxButton != null && this.brushBoxPopup != null) {
```

Comment:
`IsEnabled` can change after the template is applied.

Reply:
Yes, actually, we don't even need that test, as the button that triggers the popup is already bound to `IsEnabled` in the XAML.